cmd/gateway/commands.go

```diff
@@ -183,6 +186,7 @@ func createStaticModeCommand() *cobra.Command {
 				GatewayClassName:         gatewayClassName.value,
 				GatewayNsName:            gwNsName,
 				UpdateGatewayClassStatus: updateGCStatus,
+				DisableProductTelemetry:  disableProductTelemetry,
```

Comment:
Since there is another TelemetryReportPeriod already, would it make sense to create a struct to group related configuration options?